modules/core/src/main/scala/gem/Program.scala

```diff
@@ -1,8 +1,18 @@
 package gem
 
-case class Program(id: Program.Id, title: String)
+import scalaz._, Scalaz._
+
+case class Program[A](id: Program.Id, title: String, observations: List[A])
 
 object Program {
+
   type Id = edu.gemini.spModel.core.ProgramId
```

Comment:
For now, yes. But I think for most things it makes sense to pull them over as needed (and some things will need to be rewritten in Scala so they work in scala-js).